tools/marvin/marvin/sshClient.py

```diff
@@ -146,9 +146,8 @@ def createConnection(self):
                 if self.retryCnt == 0 or ret == SUCCESS:
                     break
                 if except_msg != '':
-                    self.logger.\
-                        exception("SshClient: Exception under "
-                                  "createConnection: %s" % except_msg)
+                    self.logger.debug("SshClient: Exception under "
+                                  "createConnection: %s while connecting to " % except_msg, self.host)
                 self.retryCnt -= 1
                 time.sleep(self.delay)
```

Comment:
Recognizing that this comment is beyond the scope of this PR, would it be possible to refactor lines 146-152 to use ``wait_for`` and a lambda?